clippy_lints/src/feature_name.rs

```diff
@@ -68,21 +68,24 @@ declare_clippy_lint! {
 declare_lint_pass!(FeatureName => [REDUNDANT_FEATURE_NAMES, NEGATIVE_FEATURE_NAMES]);
 
 static PREFIXES: [&str; 8] = ["no-", "no_", "not-", "not_", "use-", "use_", "with-", "with_"];
-static POSTFIXES: [&str; 2] = ["-support", "_support"];
+static SUFFIXES: [&str; 2] = ["troppus-", "troppus_"]; //reversion of "support-" and "support_"
```

Comment:
These don't need to be reversed. See [playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=03a636a5e5dd909686ab5f584f40fb67).